### PR TITLE
MM-28180 Cypress tests for System Console > Main Menu (6 test cases)

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
Cypress tests for MM-28180 

Resolves the following test cases for ticket MM-28180:
[MM-T909](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T909)
[MM-T910](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T910)
[MM-T911](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T911)
[MM-T912](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T912)
[MM-T913](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T913)
[MM-T914](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T914)

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28180

### CODE REVIEW

File: e2e/cypress/integration/system_console/main_menu_spec.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/main_menu_spec.js:10]: Please remove. We'll add it back once we found this spec stable for a few runs. 
```suggestion
```

File: e2e/cypress/integration/system_console/main_menu_spec.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    it('T913 About opens About modal', () => {
+        // # click to open about modal
+        cy.findByText('About Mattermost').click();
+
+        // * Verify about.mattermost.com link text has correct link destination and opens in a new tab
+        verifyLink('about.mattermost.com', 'http://about.mattermost.com/');
+
+        // * Verify server link text has correct link destination and opens in a new tab
+        verifyLink('server', 'https://about.mattermost.com/platform-notice-txt/');
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/main_menu_spec.js:68]: Thanks! (I've updated "platform" to reflect "server" in TM4J)

**Review by saturninoabril — APPROVED** [review]
Thanks @jfrerich! LGTM, except for a request to remove prod stage. Approving now.

**Review by saturninoabril — APPROVED** [review]

**Review by prapti — APPROVED** [review]
LGTM! Thank you @jfrerich!

**Pull request merged**